Assets/MixedRealityToolkit.SDK/Features/Input/Handlers/Manipulation/ManipulationHandler.cs

```diff
@@ -149,8 +151,14 @@ public RotationConstraintType ConstraintOnRotation
         /// </summary>
         public bool UseLocalSpaceForConstraint
         {
-            get => useLocalSpaceForConstraint;
-            set => useLocalSpaceForConstraint = value;
+            get => rotateConstraint != null && rotateConstraint.UseLocalSpaceForConstraint;
+            set
+            {
+                if (rotateConstraint != null)
+                {
+                    rotateConstraint.UseLocalSpaceForConstraint = value;
+                }
+            }
         }
```

Comment:
should we move those into the rotation constraint?

Reply:
xD didn't see i'm in manipulationhandler - fine this way :) 

---
In reply to: [337503342](https://github.com/microsoft/MixedRealityToolkit-Unity/pull/6321#discussion_r337503342) [](ancestors = 337503342)